### PR TITLE
allow to specify opacity

### PR DESCRIPTION
**Pull request by lenybernard**
``` twig
{% set opacity = "linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5))" %}
```

### CODE REVIEW

File: cover.html.twig [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <section class="cover{% if size|default(false) %} cover--{{ size }}{% endif %}{% if overlay|default(false) %} cover--overlay{% endif %}{% if rounded|default(false) %} cover--rounded{% endif %} {{ class|default('') }}"{% if id is defined %} id="{{ id }}"{% endif %}{% if height is defined %} style="height: {{ height }};"{% endif %}>
-    <div class="cover__bg{% if parallax|default(false) %} cover__bg--parallax{% endif %}"{% if background is defined %} style="background-image: url('{{ background }}');"{% endif %}></div>
+    <div class="cover__bg{% if parallax|default(false) %} cover__bg--parallax{% endif %}"{% if background is defined %} style="background-image: {% if opacity|default(null) %}{{ opacity ~ ', ' }}{% endif %}url('{{ background }}');"{% endif %}></div>
```
Comment (LoicGoyet) [inline cover.html.twig]: Is it not supposed to be :

```twig
style="{% if background|default(null) %}background-image: url('{{ background }}');{% endif %}{% if opacity|default(null) %}opacity: {{ opacity }};{% endif %}"
````

style property like `background-image: 0.5, url("paper.gif");` not supposed to work

**Comment by LoicGoyet** [comment]
linear-gradient is a function that requires a lot of vendor prefixes to be fully supported and it is quitte annoying to put them inline. And also, this usage of linear-gradient is quite tricky, and I think it would be better to set a new div for upfront overlays with opacity.

It could be like :

``` twig
<section class="cover{% if size|default(false) %} cover--{{ size }}{% endif %}{% if overlay|default(false) %} cover--overlay{% endif %}{% if rounded|default(false) %} cover--rounded{% endif %} {{ class|default('') }}"{% if id is defined %} id="{{ id }}"{% endif %}{% if height is defined %} style="height: {{ height }};"{% endif %}>
    {% if overlay|default(false) %}
        <div class="cover__overlay" style="{% if overlay.color|default(false) %}color: {{ color }};{% endif %}{% if overlay.opacity|default(false) %}opacity: {{ opacity }};{% endif %}"></div>
    {% endif %}

    <div class="cover__bg{% if parallax|default(false) %} cover__bg--parallax{% endif %}"{% if background is defined %} style="background-image: url('{{ background }}');"{% endif %}></div>

    ...
```

This let the possibility to set overlay's color and opacity with css or through the twig include or embed implementation.

**Comment by LoicGoyet** [comment]
@lenybernard I just made possible the two ways to add overlay, can you check it out ?

**Comment by lenybernard** [comment]
LGTM 🎊

**Pull request merged**